### DISCUSSION TITLE
Allow reusing of operation nodes in flows

### DISCUSSION THREAD

**Discussion by LevinGermann** [directus/directus#16356] [Feature Requests]
Create a flow and try to merge two different paths into one node again.
A failure and success path can't be merged into one one again. See video as an example:

https://user-images.githubusercontent.com/9347206/200571719-c3a2f3a0-937b-49fc-bca0-b68d86d48fee.mov

**Comment by licitdev** [comment]
Presently, each operation node can only be connected once. Marking this as an idea instead. 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
